core/src/cubos/debug_render.cpp

```diff
@@ -0,0 +1,183 @@
+#include <cubos/debug_render.hpp>
+
+#include <cubos/gl/render_device.hpp>
+
+#include <list>
+
+#include <glm/gtc/matrix_transform.hpp>
+#include <glm/gtx/transform.hpp>
+
+using namespace cubos;
+using namespace cubos::gl;
+
+static RenderDevice* debugRenderDevice;
```

Comment:
Maybe instead of having a bunch of static globals, we should make them static fields of the class, just to keep it more clean.

Reply:
in order for them to be part of the class they would still have to be initialized in the cpp file though

Reply:
Yeah of course, its just that for example in the Settings singleton we are declaring the global variables as static class fields, so it would be more consistent to do the same here.